Project.toml

```
@@ -13,7 +13,7 @@ Setfield = "efcf1570-3423-57d1-acb7-fd33fddbac46"
 SparseArrays = "2f01184e-e22b-5df5-ae63-d93ebab69eaf"
 
 [compat]
-AbstractMCMC = "2, 3, 4"
+AbstractMCMC = "2, 3"
```

I don't think we want to drop support for AbstractMCMC 4? Maybe rather drop support for < 4.
```suggestion
AbstractMCMC = "2, 3, 4"
```

---

Ah dammit, there was something weird going on during mergeing main.